tests/options/pugExplicitDiv/pug-explicit-div.test.ts

```diff
@@ -0,0 +1,48 @@
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { format } from 'prettier';
+import { plugin } from '../../../src/index';
+
+describe('Options', () => {
+	describe('pugExplicitDiv', () => {
+		const code: string = readFileSync(resolve(__dirname, 'unformatted.pug'), 'utf8');
+		const commonOpts = {
```

Comment:
I think you need to find a suitable typedef here. Maybe it's the second parameter of prettiers `format` function.

Reply:
Sorry for the tab and typescript lint issue.  I forgot to run `yarn lint` - in all the repos I work in it is automatic before commit and `prettier` catches most our stuff while still in the editor.  First type writing typescript as my company does not use it.